### ISSUE TITLE
508-defect-2 [SCREENREADER]: Headings SHOULD be used semantically 

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#18429]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Josh_

## User Story or Problem Statement

As a user, I want heading content to be semantically appropriate so I can navigate an outline of the page using headings easily.

## Details

When no results are found or when a form is retired, a paragraph of feedback is provided to the user. In both cases, the length 
 and content are too verbose to semantically be considered as a heading that belongs in a page outline. Since no additional content exists below these `<h2>`s, they are semantically closer to `<p>` tags.

## Acceptance Criteria

- [x] Headings are semantically and consistently used

## Environment

* Operating System: all
* Browser: all
* Screenreading device: any
* Server destination: production

## Steps to Recreate

1. Enter `<https://staging.va.gov/find-forms/?q=24-0296>` or `<https://staging.va.gov/find-forms/?q=xyz>` in browser
2. Confirm the message provided is a`<h2>`

## Proposed Solution (if known)

Two possible solutions could exist:

### Solution A: Split the `<h2>` message into a heading and a paragraph.
For example, instead of:

`<h2>No results were found for "xyz." Try using fewer words or broadening your search. If you're looking for non-VA forms, go to the GSA Forms Library.</h2>`

Split it into two pieces:

`<h2>No results were found for "xyz."</h2><p>Try using a few words...</p>`

## WCAG or Vendor Guidance (optional)

* [WCAG guidance on headings](https://www.w3.org/WAI/tutorials/page-structure/headings/)

### Before
<img width="1367" alt="Screen Shot 2021-01-12 at 3 07 00 PM" src="https://user-images.githubusercontent.com/14154792/104367729-3a782e80-54e9-11eb-9272-66ceb738c2e5.png">
<img width="1343" alt="Screen Shot 2021-01-12 at 3 22 12 PM" src="https://user-images.githubusercontent.com/14154792/104368464-19fca400-54ea-11eb-9aa4-63ff80d62209.png">

### After, Solution A
<img width="1343" alt="Screen Shot 2021-01-12 at 3 10 19 PM" src="https://user-images.githubusercontent.com/14154792/104367731-3b10c500-54e9-11eb-97a7-97cbff7ddbfe.png">

### After, Solution B
<img width="1290" alt="Screen Shot 2021-01-12 at 3 34 39 PM" src="https://user-images.githubusercontent.com/14154792/104369613-b5424900-54eb-11eb-83b2-64991f050393.png">

**Comment by MarciMcGuireGCIO** [comment]
Moving all outstanding Find a VA Form 508 issues into a single Epic to clean up backlog.

**Comment by zacharymorel** [comment]
@joshkimux This is ready for review on https://www.va.gov/find-forms/?q=121212121

**Comment by joshkimux** [comment]
Thanks @zacharymorel , closing this out

**Comment by MarciMcGuireGCIO** [comment]
Thank you @zacharymorel and @joshkimux !
